functional/adoptopenjdk/src/net/adoptopenjdk/test/AdoptOpenJDKVendorTest.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.adoptopenjdk.test;
+
+import org.testng.annotations.Test;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+/**
+ * Tests whether "AdoptOpenJDK" appears as vendor in all the places it is supposed to.
+ */
+@Test(groups = {"level.extended"})
+public class AdoptOpenJDKVendorTest {
+
+    @Test
+    public void javaVersionPrintsAdoptOpenJDK() {
+        String testJdkHome = System.getenv("TEST_JDK_HOME");
+        if (testJdkHome == null) {
+            throw new AssertionError("TEST_JDK_HOME is not set");
+        }
+
+        List<String> command = new ArrayList<>();
+        command.add(String.format("%s/bin/java", testJdkHome));
+        command.add("-version");
```

Comment:
Instead of running `java -version`, can we just the info from `System.getProperty("java.fullversion")`? If so, we can avoid running the command within java and using BufferedReader to get output.

Reply:
The purpose of this test is to actually run the command and check its output. We have regressions in this area, currently on ARM.

Reply:
Issue on ARM is this one: https://github.com/AdoptOpenJDK/openjdk-build/issues/2229, does it mean that the property returns correctly but the java -version output does not match the property?

Reply:
Properties:

```
    java.runtime.name = OpenJDK Runtime Environment
    java.runtime.version = 1.8.0_275-b01
    java.specification.name = Java Platform API Specification
    java.specification.vendor = Oracle Corporation
    java.specification.version = 1.8
    java.vendor = AdoptOpenJDK
    java.vendor.url = https://adoptopenjdk.net/
    java.vendor.url.bug = https://github.com/AdoptOpenJDK/openjdk-support/issues
    java.version = 1.8.0_275
    java.vm.info = mixed mode
    java.vm.name = OpenJDK Client VM
    java.vm.specification.name = Java Virtual Machine Specification
    java.vm.specification.vendor = Oracle Corporation
    java.vm.specification.version = 1.8
    java.vm.vendor = AdoptOpenJDK
    java.vm.version = 25.275-b01
```

`java -version`:

```
/usr/lib/jvm/jdk8/bin/java -version
openjdk version "1.8.0_275"
OpenJDK Runtime Environment (build 1.8.0_275-b01)
OpenJDK Client VM (build 25.275-b01, mixed mode)
```